pages/common/airodump-ng.md

```diff
@@ -0,0 +1,13 @@
+# airodump-ng
+
+> Capture packets and display information about wireless networks.
+> Part of `aircrack-ng`.
```

Comment:
Do you think this line should also be added to `airmon-ng`?

Reply:
Hi, well I think it should be added based on the other pages where it is specified as that. Just let me know if I can add that change to this PR.

Reply:
Sure, you can add it.